core-web/apps/dotcms-ui/src/app/view/directives/dot-avatar/dot-avatar.directive.ts

```diff
@@ -0,0 +1,48 @@
+import {
+    Directive,
+    ElementRef,
+    HostListener,
+    Input,
+    OnChanges,
+    SimpleChanges
+} from '@angular/core';
+
+import { Avatar } from 'primeng/avatar';
+
+@Directive({
+    selector: 'p-avatar[dotAvatar]',
+    standalone: true
+})
+export class DotAvatarDirective implements OnChanges {
```

Comment:
If this directive will only be used with the dot-gravatar component, I would move it to where the component is in a folder called directives/

Reply:
No, we made a whole refactor of this directive and now It's being used in several places, but that was made in another branch that is yet to be merge

Reply:
Excellent, I like reusability.